app/graphql/mutations/create_task.rb

```diff
@@ -0,0 +1,18 @@
+module Mutations
+  class CreateTask < BaseMutation
```

Comment:
Can you move this mutation to another PR please?

Reply:
done

Reply:
@DataJoshua what about removing it from this PR? :)